tests/containers/podman_integration.pm

```diff
@@ -61,12 +92,14 @@ sub run {
     parse_extra_log(TAP => $log_file);
 
     # user /remote
-    $log_file = "bats-user-remote.tap";
-    assert_script_run "echo $log_file .. > $log_file";
-    background_script_run "podman system service --timeout=0";
-    script_run "env PODMAN=/usr/bin/podman QUADLET=/usr/libexec/podman/quadlet hack/bats --rootless --remote | tee -a $log_file", 2600;
-    parse_extra_log(TAP => $log_file);
-    script_run 'kill %1';
+    if (script_run('command -v podman-remote> /dev/null') == 0) {
```

Comment:
if these conditions would fail, doesn't it mean that the test would be green in openQA without actually running the integration tests?

Reply:
> if these conditions would fail, doesn't it mean that the test would be green in openQA without actually running the integration tests?

There are 4 subtests in this module: root/local, root/remote, rootless/local & rootless/remote.  Since `podman-remote` isn't available in SLE Micro, we can't test the remote subtests.

Reply:
in that case, I would rather skip the test only for sle-micro and make it explicit.

Reply:
> in that case, I would rather skip the test only for sle-micro and make it explicit.

Mmm.  Makes sense to make it a bug if a specific flavour lacks podman-remote.  Will fix.